scss/_patterns_chip.scss

```diff
@@ -1,90 +1,168 @@
+  } @else {
+    $color-background: map-get($colors-light-theme--tinted-backgrounds, neutral);
+    $color-border: map-get($colors-light-theme--tinted-borders, neutral);
+  }
+
+  @extend %small-text;
+  @include vf-animation(#{background-color, border-color}, snap, out);
+  @include vf-focus;
+
+  background-color: $color-background;
+  border: 1px solid $color-border;
+  border-radius: $chip-line-height;
+  display: inline-flex;
+  line-height: $chip-line-height;
+  margin: map-get($nudges, p) $sph--small 0 0;
+  max-width: 100%;
+  padding-bottom: 0.25rem;
+  padding-left: $sph--large;
+  padding-right: $sph--large;
+  padding-top: 0.25rem;
+  user-select: none;
+  white-space: nowrap;
+
+  &:hover {
+    background-color: adjust-color($color-background, $lightness: -100% * $hover-background-opacity-amount, $saturation: -100% * 2 * $hover-background-opacity-amount);
+  }
+
+  &.is-selected,
+  &:active {
+    background-color: adjust-color($color-background, $lightness: -100% * $active-background-opacity-amount, $saturation: -100% * 2 * $active-background-opacity-amount);
+  }
+
+  &.is-inline {
+    margin: 0;
+  }
+
+  .p-chip__lead,
+  .p-chip__value {
+    margin-bottom: 0;
+    overflow: hidden;
+    text-overflow: ellipsis;
+  }
+
+  .p-chip__lead {
+    $space-between-lead-and-value: 0.25rem;
+    @extend %muted-text;
+    @extend %x-small-text;
 
-    background-color: $color-background-base;
-    border-radius: $chip-line-height;
-    display: inline-flex;
     line-height: $chip-line-height;
-    margin: map-get($nudges, p) $sph--small 0 0;
-    max-width: 100%;
-    padding-bottom: 0.25rem;
-    padding-left: $sph--large;
-    padding-right: $sph--large;
-    padding-top: 0.25rem;
-    user-select: none;
-    white-space: nowrap;
-
-    &:hover {
-      background-color: $color-background-hover;
-    }
+    padding-right: $space-between-lead-and-value * 2;
+    padding-top: 0.1rem;
+    position: relative;
+    text-transform: uppercase;
 
-    &.is-selected,
-    &:active {
-      background-color: $colors--light-theme--background-active;
+    &::after {
+      content: '\00a0:';
+      position: absolute;
+      right: $space-between-lead-and-value;
     }
+  }
 
-    .p-chip__lead,
-    .p-chip__value {
-      margin-bottom: 0;
-      overflow: hidden;
-      text-overflow: ellipsis;
-    }
+  .p-chip__badge {
+    @extend %vf-p-chip-badge-base;
+
+    background-color: $color-x-dark;
+  }
 
-    .p-chip__lead {
-      $space-between-lead-and-value: 0.25rem;
-      @extend %muted-text;
-      @extend %x-small-text;
-
-      line-height: $chip-line-height;
-      padding-right: $space-between-lead-and-value * 2;
-      padding-top: 0.1rem;
-      position: relative;
-      text-transform: uppercase;
-
-      &::after {
-        content: '\00a0:';
-        position: absolute;
-        right: $space-between-lead-and-value;
-      }
+  .p-chip__badge--negative {
```

Comment:
I think we talked about dropping badges from this PR?

Reply:
true, dropped it now